api/controllers/lectureController.js

```diff
@@ -0,0 +1,12 @@
+const { lectureService } = require("../services");
+const { catchAsync, CustomError } = require("../utils/error");
+
+
+const  getLectureByLectureId = catchAsync ( async (req, res) => {
+    const lectureId = req.params.lectureId;
+    if(!lectureId) throw new CustomError( 'BAD REQUEST', 400)
+    const detail = await lectureService.getLectureByLectureId(lectureId);
+    return res.status(200).json({ data: detail });
+});
```

Comment:
가독성을 위해 if나 for가는 새로구 구문들 사이에 공백을 추가해 주면 좋을 것 같습니다.
```suggestion
const  getLectureByLectureId = catchAsync ( async (req, res) => {
    const lectureId = req.params.lectureId;
    
    if(!lectureId) throw new CustomError( 'BAD REQUEST', 400)
    
    const detail = await lectureService.getLectureByLectureId(lectureId);
    return res.status(200).json({ data: detail });
});
```